pkg/activator/handler/concurrency_reporter.go

```diff
@@ -46,6 +47,7 @@ const reportInterval = time.Second
 type revisionStats struct {
 	stats        *network.RequestStats
 	firstRequest float64
+	refs         *atomic.Int64
```

Comment:
nit: think this doesn't need to be a pointer (see e.g. https://github.com/knative/serving/blob/c00c6d55135507b085f19ff4ea44e1cfb36060c4/vendor/knative.dev/pkg/metrics/metrics.go#L70 and https://github.com/knative/serving/blob/772caeca629b2218d7612685a2cc0f09bbed490d/pkg/queue/breaker.go#L51)